plone/app/tiles/tests/test_forms.py

```diff
@@ -97,7 +101,7 @@ def set(self, value):
             self.browser.getControl(label="Save").click()
 
             # Set should have been called three times, once for each field
```

Comment:
Nitpicking: comments should be updated 😉

Reply:
Fair point. :-)
I will first merge, and then do an extra commit to fix the comments.

Reply:
Fixed in commit 5716d075528d2bca146c192cda316d4e2390a113.